Donation-platform/src/main/java/com/google/DbCalls.java

```diff
@@ -126,6 +126,23 @@ public Charity getCharityByName(String name) throws Exception {
     // Return all charity objects containing tag.
     return charityDataStore;
   }
+  // Function returns all charities with one of the three specified tags.
+  public Collection<Charity> getCharitiesByTags(String name1, String name2, String name3) throws Exception {
+    ArrayList<Charity> charityDataStore = new ArrayList<Charity>();
```

Comment:
rename it to personalizedCharitiesList

Reply:
or something more meaningful

Reply:
I understand the use of this function and I think it's okay as is, but is this good practice to make functions that are basically the same thing but for specific use cases? Does this take away from the principle of abstraction in general? What do you think?

Reply:
I have also made significant changes to this file in my branch, but I will take care of the integration.

Reply:
Done.